app/views/NirmsQuestionView.scala.html

```diff
@@ -0,0 +1,52 @@
+@*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@this(
+    layout: templates.Layout,
+    formHelper: FormWithCSRF,
+    heading1: components.Heading1,
+    heading2: components.Heading2,
+    paragraph: components.Paragraph,
+    govukRadios: GovukRadios,
+    govukButton: GovukButton,
+    link: components.Link
+)
+
+@(form: Form[Boolean])(implicit request: Request[_], messages: Messages)
+
+@layout(
+    pageTitle    = titleNoForm(messages("nirmsQuestion.title")),
+    showBackLink = true
+) {
+    @heading1(messages("nirmsQuestion.h1"))
+    @paragraph(Html(messages("nirmsQuestion.p1")))
+    @paragraph(Html(messages("nirmsQuestion.p2", link(messages("nirmsQuestion.p2.linkText"),"https://www.gov.uk/government/publications/retail-movement-scheme-how-the-scheme-will-work/retail-movement-scheme-how-the-scheme-will-work#product-eligibility"))))
+
+    @heading2(messages("nirmsQuestion.h2"))
+
+    @formHelper(action = NirmsQuestionController.onSubmit) {
+        @govukRadios(
+          RadiosViewModel.yesNo(
+                field = form("value"),
+                legend = Legend()
```

Comment:
The Legend shouldn't be empty, we can use `nirmsQuestion.h1` for this

Reply:
That would change the design so it no longer looks like the prototype though? 

with the legend
<img width="664" alt="image" src="https://github.com/hmrc/trader-goods-profiles-frontend/assets/114592674/0676e83f-7ea8-4938-a017-5dcc03a42261">

vs prototype
<img width="664" alt="image" src="https://github.com/hmrc/trader-goods-profiles-frontend/assets/114592674/d2de2ab1-98d4-4274-a4f3-cc91e7aed7e1">

Reply:
I think there’s a way to hide it, .asHiddenSomething
I recall it being something we had to update in a prior project for accessibility reasons

Reply:
ok yeah while looking up how to do it I found the accessibility repo saying how its a common accessibility error! 

Updated :)